### PR TITLE
Update measures

### PR DESCRIPTION
**Pull request by JulianBlohm**
- Update measures
- Add date of change

### CODE REVIEW

File: src/components/Measures/GeneralMeasures.js [hunk 1]
```diff
@@ -89,6 +89,10 @@ export const GeneralMeasures = ({ riskLevel }) => {
         jedoch, dass pro Bundesland und Landkreis diese teilweise abweichen
         können.
       </p>
+      <p>
+        Alle bis zum 31. Januar befristeten Maßnahmen sind vorläufig bis zum 14.
```
Comment (jonicious) [inline src/components/Measures/GeneralMeasures.js:93]: I thiiink, this component is generally unused right now. Does not hurt to add this there but I think it won't be shown.

File: src/components/Measures/LockdownMeasures.js [hunk 2]
```diff
@@ -34,6 +39,11 @@ const measures = [
     description:
       'Restaurants sowie Bars, Clubs, Diskotheken, Kneipen und ähnliche Einrichtungen bleiben geschlossen. Davon ausgenommen sind die Lieferung und Abholung von Speisen für den Verzehr zu Hause.',
   },
+  {
+    title: 'Homeoffice',
+    description:
+      'Arbeitgeber*Innen müssen künftig überall, wo es möglich ist, Arbeiten im Homeoffice ermöglichen.',
```
Comment (jonicious) [inline src/components/Measures/LockdownMeasures.js:45]: Shouldn't it be "Arbeitgeber*innen"? I am not sure but I think I heard that : is better for screen readers, so "Arbeitgeber:innen". Totally okay if we keep it like this though!

**Comment by jonicious** [comment]
Nice, thanks for taking care of it! 

Personally, I would maybe rearrange the measures since the very two last lines look a bit weird.

**Review by jonicious — APPROVED** [review]

File: src/components/Measures/GeneralMeasures.js (hunk 1 above)
Comment (JulianBlohm) [reply src/components/Measures/GeneralMeasures.js:93]: Recognized it after changing, could'nt know.

**Pull request merged**